src/Linters/UnusedParameterLinter.hack

```diff
@@ -25,15 +25,26 @@ final class UnusedParameterLinter extends AutoFixingASTLinter {
       return null;
     }
 
-    $name = $node->getName();
-    if (!$name is VariableToken) {
+    $name_node = $node->getName();
```

Comment:
For a future change, I think this is roughly where we want to check that $node is a child of $functionish, not  any descendant to fix #341